### PR TITLE
Added Customization Features 

### PR DESCRIPTION
**Pull request by SurajLad**
Hello, 
I have Added various UI Related Improvements/Features in your existing plugin, Updated ReadMe File Containing Details Of All New things Added Like, : 

- Added More Features Like Title/Hint as Widget, Chip Customizations(color/label)
- Dialog Customizations - Dialog textStyle
Find More details on readMe.md

### CODE REVIEW

**Comment by RohanSakhale** [comment]
@cetorres can you please review this, these are good features and provides various ways of customisation for existing library

**Comment by cetorres** [comment]
I'll review it, thanks.

File: lib/multiselect_dialog.dart [hunk 1]
```diff
@@ -53,7 +66,8 @@ class _MultiSelectDialogState<V> extends State<MultiSelectDialog<V>> {
   @override
   Widget build(BuildContext context) {
     return AlertDialog(
-      title: Text(widget.title),
+      title: widget.title,
```
Comment (RohanSakhale) [inline lib/multiselect_dialog.dart:69]: Having this referenced from outside gives more control for customization as and when required

**Comment by cetorres** [comment]
Hi @SurajLad,
I merged your code and published the new version of the package to Pub.Dev.
Thank you!

**Pull request merged**